src/en/guide/get_started/setup.md

```diff
@@ -44,12 +44,115 @@ Remember: If you follow the `git clone` approach in [Installation](./installatio
 
 Here you are good to go! See [Quickstart](./quickstart) or our [Tutorials](/guide/tutorials/agent_101) for your first run!
 
-## Setup for different model providers
 
-### OpenAI
+## Detail Setting
+Some future projects may require other API models, so you should configure the keys of other Apis according to your needs.
```

Comment:
MetaGPT supports various LLM models, configure the keys of model APIs according to your needs.